client/src/pages/WeeklyTable.js

```diff
@@ -2,21 +2,26 @@ import React from "react";
 
 import TableHeading from "./TableHeading";
 
-const WeeklyTable = (props) => {
-	return (
-		<div>
-			<table className="table">
-				<TableHeading />
-				<tbody>
-					{props.result.map((info, index) => (
-						<tr key={index}>
-							<td>{info.username}</td>
-						</tr>
-					))}
-				</tbody>
-			</table>
-		</div>
-	);
-}
+const fillInRow = (seat, bookings) =>
+	bookings
+		.filter((booking) => booking.desk_user_id === seat)
+		.map((b, index) => (
+			<td key={index}>
+				{b.username}
+			</td>
+		));
+
+const WeeklyTable = (props) => (
+	<div>
+		<table>
+			<TableHeading />
```

Comment:
I think the TableHeading is missing in your commit?

I would actually hardcode a week and map that week <DayHeading date={day}> -> reasoning, it will really help with creating a button to create a new booking for that day if the day is a prop for the whole heading